test/jest/Go/boardState.test.ts

```diff
@@ -13,7 +13,8 @@ describe("Board analysis utility tests", () => {
       .filter((p) => p === "O").length;
     expect(whitePieceCount).toEqual(1);
     expect(result).toEqual({
-      board: expect.any(Object),
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      board: expect.any(Object), // This board state is different every run
```

Comment:
Can't you use `as` here instead of disabling the lint?

Reply:
I changed this to not need the .any() here and better test the actual value of result.board in the next expect()